great_expectations/util.py

```diff
@@ -1199,3 +1199,11 @@ def get_sqlalchemy_domain_data(domain_data):
     # engine.get_domain_records returns a valid select object;
     # calling fetchall at execution is equivalent to a SELECT *
     return domain_data
+
+
+def import_make_url():
+    if version.parse(sa.__version__) < version.parse("1.4"):
+        from sqlalchemy.engine.url import make_url
```

Comment:
Would you mind adding a comment about deprecation here (just to be consistent with the other helper functions we have in this file). I really like what you did for `get_sqlalchemy_selectable` but even something as simple as what we have for `get_sqlalchemy_url` is sufficient.